### ISSUE TITLE
Should we use GitPoll to vote on syntax and features for Streem?

### ISSUE THREAD

**Issue by alexispurslane** [matz/streem#52]
It seems that certain features and changes to Streem are popular or unpopular, but wouldn't it be better to actually manage polls/votes on certain things to decide, instead of doing it by hand?

http://poll.gitrun.com/

**Comment by felixangell** [comment]
This is a 2 year old issue, but technically you could just use the thumbs up/down responses on GitHub issues for this instead.

**Comment by tbodt** [comment]
@felixangell they weren't there 2 years ago...

**Comment by felixangell** [comment]
Yeah you're right. I'm just pointing out that this issue is now technically irrelevant if everyone doesn't mind using the response system on GitHub.

**Comment by nickmccurdy** [comment]
It seems like gitpoll lets you have more than two options, but it probably isn't necessarily. I vote to close this.
